src/tooltip.tsx

```diff
@@ -105,6 +103,9 @@ type Props = {
     targetLabel: string;
     delay?: boolean;
     dataAttributes?: DataAttributes;
+    fullWidth?: boolean;
```

Comment:
I'm not sure why the tooltip should resize its target 🤔

Reply:
the resizing was necessary to align the Tooltip with the thumb of the component slider.
 Without the redirect, the tooltip would extend beyond the width of the new Slider component.

![image](https://github.com/Telefonica/mistica-web/assets/123959276/dbf56094-ca3e-4d4a-bad1-f1ac233ba789)

Reply:
why doesn't the slider component handle this with the tooltip?